docs/source/guides/aws_sagemaker_end-to-end_workflow.rst

```diff
@@ -0,0 +1,475 @@
+
+.. code-block:: python
+
+    from bentoml_service import Service
+
+    #create a service instance for the movie review model
+    bento_svc = Service()
+
+    # pack model artifacts
+    bento_svc.pack('model',model)
+    bento_svc.pack('tokenizer',tokenizer)
+
+    #save the prediction service for model serving 
+    saved_path = bento_svc.save()
+
+.. code-block:: python
+
+    # sample output
+
+    INFO:tensorflow:Assets written to: /private/var/folders/vn/bytl5x0n3vgg1vmg7n6qkqtc0000gn/T/bentoml-temp-35n_doz7/Service/artifacts/model_saved_model/assets
+    [2020-06-25 19:57:01,302] INFO - Detect BentoML installed in development model, copying local BentoML module file to target saved bundle path
+    running sdist
+    running egg_info
+    writing BentoML.egg-info/PKG-INFO
+    writing dependency_links to BentoML.egg-info/dependency_links.txt
+    writing entry points to BentoML.egg-info/entry_points.txt
+    writing requirements to BentoML.egg-info/requires.txt
+    writing top-level names to BentoML.egg-info/top_level.txt
+    reading manifest file 'BentoML.egg-info/SOURCES.txt'
+    reading manifest template 'MANIFEST.in'
+    ...
+    ...
+    UPDATING BentoML-0.8.1+0.g5b6bd29.dirty/bentoml/_version.py
+    set BentoML-0.8.1+0.g5b6bd29.dirty/bentoml/_version.py to '0.8.1+0.g5b6bd29.dirty'
+    Creating tar archive
+    removing 'BentoML-0.8.1+0.g5b6bd29.dirty' (and everything under it)
+    [2020-06-25 19:57:03,142] WARNING - Saved BentoService bundle version mismatch: loading BentoService bundle create with BentoML version 0.8.1,  but loading from BentoML version 0.8.1+0.g5b6bd29.dirty
+    [2020-06-25 19:57:03,958] INFO - BentoService bundle 'Service:20200625195616_62D0DB' saved to: /Users/amy/bentoml/repository/Service/20200625195616_62D0DB
+
+
+=================================
+Upload BentoService Bundle to S3
+=================================
+
+As mentioned earlier, BentoML also provides ways to change the model repository - allowing data science teams to share the BentoService bundle easily for better collaborations. One way is by uploading it to the cloud services such as AWS S3. Using the same scripts as above and passing the S3 bucket URL into  :code:`.save()` , it will deploy the BentoService bundle directly into the S3 movie-review-dataset bucket we created earlier.
+
+.. code-block:: python
+
+    from bentoml_service import Service
+
+    #create a service instance for the movie review model
+    bento_svc = Service()
+ 
+    # pack model artifacts
+    bento_svc.pack('model',model)
+    bento_svc.pack('tokenizer',tokenizer)
+
+    #save the prediction service to aws S3
+    saved_path = bento_svc.save(''s3://movie-review-dataset/'')
```

Comment:
Let's not use this deprecated API to save directly to s3.  Take a look at the documentation on YataiService here: https://docs.bentoml.org/en/latest/concepts.html#model-management

Ideally, the user deploy a YataiService server that's shared with the entire data science team, and the YataiService is configured to save files to s3.  The end-user can then configure BentoML to use remote YataiService, which makes `save` talking to the remote YataiService and upload files to s3.